table.gs

```diff
@@ -421,6 +421,21 @@ Table.prototype.getItemById = function (itemId) {
 };
 
 
+/**
+ * Method to return only distinct different values in a field.
+ */
+Table.prototype.distinct = function(field) {
+  var list = [];
+  for (var i = 0; i < this.items.length; i++) {
+    list.push(this.items[i].getFieldValue(field));
+  }
+  var unique = list.filter(function(value, index, self) { 
+    return self.indexOf(value) === index;
```

Comment:
It's a really cool "hack" to see if the item is unique, but could we add a comment explaining it? It can be confusing